pysquared/logger.py

```diff
@@ -18,8 +18,6 @@
 import traceback
 from collections import OrderedDict
 
-import adafruit_pathlib
```

Comment:
While I previously suggested using `adafruit_pathlib`, this would have caused additional pain in a few ways:
- All CircuitPython functions that interact with the filesystem expect a string path which would mean that we would always need to output the path to a string anyways.
- `adafruit_pathlib` did not have any `join()` semantics which made adding to a `Path` object clunky. (this could be submitted upstream)
- Most modules in pysquared that use the `Logger`, I found that I would have had to mock away the use of `adafruit_pathlib` in many many test files.

So ultimately, I removed it.